lua/entities/gmod_wire_expression2/core/e2lib.lua

```diff
@@ -943,6 +946,7 @@ hook.Add("InitPostEntity", "e2lib", function()
 				local owner = entity:CPPIGetOwner()
 				if IsValid(owner) then return owner end
 
+				if not getOwnerEnabled then return end
```

Comment:
I believe this check is unnecessary. The E2Lib function is already guarded against using `Entity.GetOwner`. I don't think removing this fallback behavior, as unlikely as it is, is meaningful.

Reply:
Woops yeah sorry, that one wasn't intended and was purely used during testing. I've removed it